fakeData.py

```diff
@@ -0,0 +1,135 @@
+import numpy as np
+import random
+from faker import Faker
+import pandas as pd
+
+"""
+
+Copmpile a fake dataset resembling the dataset Michael collected on his trip to Guatemala in.
+
+The fake dataset uses relative proportions of the tree types in the original dataset as 
+probabilities for selection of the tree types of this dataset. So, although it is not the original data--
+and the names are fake!--it resembles the patterns of the co-op.
+
+"""
+
+fake = Faker('es_MX') # faker does not have options for any American Spanish varieties besides Mexico,
+# so we select Mexico because (1) you can find coffee plantations in Mexico, and (2) it is just north of Guatemala
+
+numFarmers = 40 # how many farms/farmers we want in our fake dataset
+
+
+# list the species of trees plated on the co-op
+treeNames = ['borbon', 'catuai', 'e14', 'catura']
+
+# utilize probabilities of original dataset
+freqProb = {'borbon': 0.5534, 'catuai': 0.4191, 'e14': 0.0046, 'catura': 0.02299} 
+
+# even if order changes, the probabilities still correspond to the strings
+#lsTreeTypes = random.choices()
+
+
+lsOfFarmers = [] # create a list to append these farmers to
+lsOfTrees = [] # create a list to hold lists for each farmer's trees
+lsOfCuerdas = [] # creat a list to hold the number of cuerdas
+lsOfAges = []
+
+for i in range(numFarmers):
+    lsOfFarmers.append(fake.name()) # append the fake name, who is the owner of the farm
+    upperBounds = 4 # max num of tree types a farmer may have
+    lowerBounds = 1 # min num of tree types a farmer may have
+    numStrains = random.randint(lowerBounds,upperBounds) # decide how many tree types that one farmer will own
+    
+    while True:
+        # create a trained but random list of farmer's trees weighted by the results from Guatemala
+        # assure k is large enough that we're guarenteed to get 4 unique values without iterating too many times
+        sampleLs = random.choices(treeNames, weights = (freqProb[treeNames[0]], freqProb[treeNames[1]],
+                                                  freqProb[treeNames[2]], freqProb[treeNames[3]]), k = 500)
+        
+        sampleSet = set(sampleLs)
+        
+        if len(sampleSet) >= upperBounds:
+            break
+            
+    
+    
+    # sample from this list to the size (also prevents repeats)
+    treeLs = []#random.sample(sampleLs, numStrains)
```

Comment:
these are trees that belong to a particular farmer, right?

I think we can use `_trees = ` kind of pattern for throwaway variables and make them a bit more readable. Don't assume `Ls` is understood as an abbreviation. small things like this seem like nitpicks but will help a lot with future maintainability of code. 

however, don't take that to mean "do this now" for this PR to be accepted. If it works, it's good enough for this iteration. We can/should raise an issue to track a separate effort to "refactor" this function to make it more readable (which will look like a quick sit down, find+replace.. maybe good practice for vim...make sure tests still pass, then very quick to review PR to merge it in).